tests/integ/test_telemetry.py

```diff
@@ -930,6 +930,8 @@ def add_one(session_, x):
 
 @pytest.mark.udf
 def test_udtf_call_and_invoke(session, resources_path):
+    sql_simplifier_enabled = session.sql_simplifier_enabled
+    session.sql_simplifier_enabled = True
```

Comment:
this test has order of telemetry reversed in sql simplifier disabled case. I'm adding this change here to test fix that.

Reply:
Could you be more explicit on the error message? From a quick glance our assertion is on `dict` being equal and I think `dict` is not sensitive to ordering.

Reply:
The telemetry message log is a list of dict of dicts. In our tests, we extract a specific index using `extract_telemetry_log_data` using `index` like `-1` or `-2` saying select the last or second last entry of the message log.

When I test this without simplifier enabled, the order of telemetry messages in the message log is flipped but the message is present in both cases. I'm not sure if this is the best way to fix this error though - I am open to more suggestions here.

Reply:
updated based on your suggestion. This looks much better now